modules/custom/advanced_help_block/config/install/field.field.advanced_help_block.advanced_help_block.field_ahb_video.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.advanced_help_block.field_ahb_video
+  module:
+    - advanced_help_block
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_ahb_video
+id: advanced_help_block.advanced_help_block.field_ahb_video
+field_name: field_ahb_video
+entity_type: advanced_help_block
+bundle: advanced_help_block
+label: Video
+description: ''
```

Comment:
Could you fill in all descriptions for the end users?

Reply:
done.